impl/src/main/java/io/kroxylicious/testing/kafka/testcontainers/TestcontainersKafkaCluster.java

```diff
@@ -330,7 +331,22 @@ public synchronized void removeBroker(int nodeId) throws UnsupportedOperationExc
         ports.values().forEach(pm -> pm.remove(nodeId));
 
         var kafkaContainer = brokers.remove(nodeId);
-        kafkaContainer.stop();
+        // Note that GenericContainer#stop actually implements stop using kill, so the broker doesn't have chance to
```

Comment:
Need to raise a TC issue about this.

Reply:
It already exists: https://github.com/testcontainers/testcontainers-java/issues/1000

Reply:
I wonder if we should actually do what they suggest in https://github.com/testcontainers/testcontainers-java/issues/1000#issuecomment-840657453 and call graceful stop ourselves rather than killing? (Entirely happy for that to be a separate PR)